pandas/core/frame.py

```diff
@@ -4125,16 +4125,23 @@ def combine_first(self, other):
         ----------
         other : DataFrame
 
+        Returns
+        -------
+        combined : DataFrame
+
         Examples
         --------
-        a's values prioritized, use values from b to fill holes:
-
-        >>> a.combine_first(b)
+        df1's values prioritized, use values from df2 to fill holes:
 
+        >>> df1 = pd.DataFrame([[1, np.nan]])
+        >>> df2 = pd.DataFrame([[3, 4]])
+        >>> df1.combine_first(df2)
+           0    1
+        0  1  4.0
 
-        Returns
-        -------
-        combined : DataFrame
+        See Also
+        --------
+        DataFrame.combine
```

Comment:
Can you put a short sentence here highlighting the difference?

Reply:
Added